Test.java

```diff
@@ -168,9 +168,9 @@ public void consume(String line) {
 				kubectl.clearArgs();
 				kubectl.addArgs("get", "service", "onedev", "-o", 
 						"jsonpath={range .spec.ports[*]}{.name} {.port}{'\\n'}{end}");
-				AtomicReference<Integer> httpPortRef = new AtomicReference<>(null);
-				AtomicReference<Integer> httpsPortRef = new AtomicReference<>(null);
-				AtomicReference<Integer> sshPortRef = new AtomicReference<>(null);
+				int httpPort = 0;
+				int httpsPort = 0;
+				int sshPort = 0;
```

Comment:
int might be overflow. suggest to use long
```suggestion
				long httpPort = 0;
				long httpsPort = 0;
				long sshPort = 0;
```